src/ldeether.c

```diff
@@ -103,7 +103,7 @@ int main(int argc, char *argv[]) {
     /* Use DLPI to connect to the ethernet.  This code is stolen
        from NFSWATCH4.3
     */
-    char *etherdev = (char *)getenv("LDEETHERDEV");
+    char *etherdev = getenv("LDEETHERDEV");
```

Comment:
Missing the <code>#include <stdlib.h></code>

Reply:
Fixed and force pushed.